### PR TITLE
BST Traversals in C++ Added!

### PR DESCRIPTION
**Pull request by yashikajotwani12**
#144 Added BST traversals in C++
@plazzy99 
->2 testcases
->Explained using comments
->time and space complexity
->successfully compiled

### CODE REVIEW

**Comment by yashikajotwani12** [comment]
@plazzy99 , please review this .

File: Code/C++/bst_traversals.cpp [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+   1. Traverse the left subtree
+   2. Traverse the right subtree
+   3. Visit the root.
+*/
+
+#include <iostream> 
+using namespace std; 
+
+struct Node
```
Comment (rudrakshi99) [inline Code/C++/bst_traversals.cpp:24]: Don't use struct in c++. Use class to implement.

File: Code/C++/bst_traversals.cpp [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+testcase 2:
+output:
+Preorder traversal of binary tree is 
+1 2 4 5 3
+Inorder traversal of binary tree is
+4 2 5 1 3
+Postorder traversal of binary tree is
+4 5 2 3 1
+
+
```
Comment (rudrakshi99) [inline Code/C++/bst_traversals.cpp]: add space and time complexity.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Code/C++/bst_traversals.cpp (hunk 1 above)
Comment (yashikajotwani12) [reply Code/C++/bst_traversals.cpp:24]: ok !! I would change it. Thanks for reviewing the code.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
you cannot solve multiple ques in same PR.

**Pull request closed without merging**